frigate/output/birdseye.py

```diff
@@ -750,6 +781,16 @@ def __init__(
 
         self.converter.start()
         self.broadcaster.start()
+        self.monitor_thread = threading.Thread(
+            target=self.monitor_enabled_states, daemon=True
+        )
+        self.monitor_thread.start()
+
+    def monitor_enabled_states(self):
```

Comment:
I don't think we need this as this concern will be addressed by the changes to the output process to handle when all cameras are disabled

Reply:
Great, I'll just remove this for now.